.pre-commit-config.yaml

```diff
@@ -95,6 +95,11 @@ repos:
         types: [python]
         require_serial: true
         files: ^homeassistant/.+\.py$
+      - id: pylint
+        name: pylint
+        entry: pylint
+        language: system
+        types: [python]
```

Comment:
You could try

```suggestion
      - id: pylint
        name: pylint
        entry: script/run-in-env.sh pylint
        language: script
        types: [python]
        require_serial: true
```

Reply:
i've already pushed a new commit fc0917e meanwhile, which is quiet equal to your suggestion 🤔